tests/product/constants.py

```diff
@@ -27,8 +27,4 @@
 DEFAULT_DOCKER_MOUNT_POINT = '/mnt/presto-admin'
 DEFAULT_LOCAL_MOUNT_POINT = os.path.join(main_dir, 'tmp/docker-pa/')
 
-BASE_IMAGE_NAME = 'jdeathe/centos-ssh'
-BASE_IMAGE_TAG = 'centos-6-1.2.0'
-
 BASE_TD_IMAGE_NAME = 'teradatalabs/centos6-ssh-test'
```

Comment:
It needs to go away, but it's going to happen in the upcoming "make  _ensure_docker_containers_started opt-out rather than opt-in" PR. For the moment _ensure... still refers to it.